### DISCUSSION TITLE
Basic dark mode support

### DISCUSSION THREAD

**Discussion by viking1304** [foundation/foundation-emails#1142] [General]
Since some people are unsure how to prepare newsletters that properly support dark mode using this framework, I am working on basic dark-mode support that everyone can use in their projects with almost no additional work.

I would like to get some help from a professional designer to choose appropriate colors for some elements used in example newsletters. Any volunteers?

I think the first three look decent, but I am not too happy with the mid-dark gray parts of the last one.

<img src="https://user-images.githubusercontent.com/28989870/228834652-fb4b9778-8619-47d6-8e38-c9f9be359e9f.png" width="300">   <img src="https://user-images.githubusercontent.com/28989870/228834661-ec8bcccf-256f-44c3-b5b5-a86902c90da7.png " width="300">

<img src="https://user-images.githubusercontent.com/28989870/228834670-aecd883b-e36e-45f2-ace0-49a82a3ed208.png " width="300">   <img src="https://user-images.githubusercontent.com/28989870/228834681-00a0827c-ff73-4290-81a6-1ebd80b821b2.png " width="300">

It's even worse with some other example templates.

I do excessive testing on many different clients and devices, but I do not work on newsletter projects too often to justify the need for the testing service. So, I would also appreciate help from someone with access to some decent newsletter testing service who could provide screenshots for specific mail clients, so we can decide what should be fixed and how.

Disclaimer: _This will not be a bulletproof solution that will work on all mail clients, but it will give everyone decent dark-mode support on most mail clients. I will avoid using hacks like liner gradient backgrounds, blend modes, and webkit text fill color since those hacks are inconsistent and must be carefully used. When using those hacks, it is very easy to get unreadable mail in some mail clients._

**Comment by iLiketurtles3000** [comment]
Nice, 
So soon we might have dark mode functionality then if you get merged. Sorry I am not a designer but just a thought. Maybe look at the color schemes utilised by web extensions such as 'midnight-lizard' that offer dark mode and see what coloring profile they pair up and use. This could be a starting point at least. Goodluck

I suggest if you
